app/src/main/java/org/mifos/selfserviceapp/api/services/ClientService.java

```diff
@@ -16,15 +17,23 @@
  */
 public interface ClientService {
 
+    String CLIENT_ID = "clientId";
```

Comment:
Ahh My bad, I didn't realize it was an interface. 

Old habits of writing Retrofit Services as inner classes :( 

Sorry about this!